src/modules/admin/components/formRules/FormRule.tsx

```diff
@@ -130,10 +140,14 @@ const FormRule: React.FC<Props> = ({ rule, formRole }) => {
           </>
         )}
       </Typography>
-      <ActiveChip
-        active={active || false}
-        // TODO(#6289) - Deletion should deactivate. Inactive rules should not be shown/returned
-      />
+      <IconButton
+        onClick={() => deactivate()}
+        size='small'
+        sx={{ height: '28px' }}
+        disabled={loading}
```

Comment:
There's a moment after the deletion completes that the button becomes enabled again, when it's no longer loading.
You could disable this on `called` instead so it remains disabled https://www.apollographql.com/docs/react/data/mutations/#mutationresult-interface-called